### DISCUSSION TITLE
How can I payout my sponsorship payments?

### DISCUSSION THREAD

**Discussion by kms0219kms** [community/community#138682] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Team,

I would like to withdraw the donations I received through my @lunaiz-corp organization.
However, I can't find the expected payment date on my dashboard and Stripe Express website.

I read the policy and it says that the first donation payment requires at least 60 days,
and since my Stripe account is currently in manual payment, according to Stripe documentation, I need to wait at least 90 more days to receive the donation (holding in Stripe).

Am I wrong here? How can I withdraw it sooner than waiting 150+ days?

Please reply here or email me at minsu.kim@lunaiz.com.

Kind regards,
Minsu Kim @ Lunaiz

**Comment by kms0219kms** [comment]
cc with @Dev-SSS1267

**Comment by kms0219kms** [comment]
Support ticket has been opened #3000118

**Reply by ebndev** [reply, marked as answer]
Hi @kms0219kms,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
